Makefile

```diff
@@ -36,7 +37,7 @@ GO_BUILD_FLAGS = -v
 GO_BUILD_LDFLAGS = -X main.version=$(VERSION)
 GQL_GEN = $(GO_PATH)/bin/gqlgen
 
-PLATFORMS := windows linux darwin
+PLATFORMS := linux darwin
```

Comment:
Why no windows? Is the windows not supported due to queue implementation?

Reply:
yes, mmap is not supported in windows , I found that there were many mmap go lib problems

Reply:
The MMap Go is largely unsupported due to the underlying system command

Reply:
Does that block compiling and running if we don't activate mmap? AFAIK, mmap is only a queue option, we could add document about it, says, only support memory-queue or other implementation in windows. Don't like the idea to exclude window directly.

Reply:
@EvanLjp I think this has not been resolved. Could you update the status?

Reply:
> @EvanLjp I think this has not been resolved. Could you update the status?

What's the resolution?

Reply:
https://github.com/apache/skywalking/issues/6046

Reply:
support memory queue  to windows